### PR TITLE
Issue #33: Evaluation script

### PR DESCRIPTION
**Pull request by shamith2**
script to evaluate a policy against another as well as analyze how a team has won the CtF

### CODE REVIEW

**Comment by skim0119** [comment]
Pull the Released branch and resolve conflicts

File: cap_eval.py [hunk 1]
```diff
@@ -65,28 +69,39 @@
...
         
         env.reset()
         game_finish = False
         
-        red_score.append(abs(reward)) if reward < 0 else blue_score.append(reward)
-        print("# Steps: %d, Time: %.2f s, blue team score: %.2f, red team score: %.2f" %
-             (steps, (time.time() - start_time), np.asarray(blue_score).mean(), np.asarray(red_score).mean()))
-        print("The Result is: %s team win by capturing %s teams's flag and by killing %s team" % (result, flag, die))
+        if reward > 0: blue_score.append(reward)
+        #print("blue team mean score: %.2f, std.dev. = %.2f" %(np.mean(blue_score), np.std(blue_score))
```
Comment (skim0119) [inline cap_eval.py]: Remove the line

**Comment by skim0119** [comment]
Look into tqdm to put progress bar for large number of episode run.

File: cap_eval.py (hunk 1 above)
Comment (shamith2) [reply cap_eval.py]: Removed comments

**Comment by shamith2** [comment]
> Look into tqdm to put progress bar for large number of episode run.
>>>>> added tqdm progress bar for episodes

**Review by skim0119 — CHANGES_REQUESTED** [review]
Could you append on README.md brief explanation of how to use this script?
>>> I updated README.md to include policy evaluation

**Review by skim0119 — APPROVED** [review]
cap_eval.py test finished

**Pull request merged**